Rakefile

```diff
@@ -37,6 +37,10 @@ if defined?(RSpec)
       t.pattern = "spec/processor_spec.rb"
     end
 
+    RSpec::Core::RakeTask.new(:detect_os) do |t|
+      t.pattern = "spec/helper/detect_os/*_spec.rb"
+    end
+
```

Comment:
Remove this fix. This is need to run tests, but don't need to be included in this PR.